js/custom.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Created by arjun on 5/24/2016.
+ */
+
+var i=0;
+window.ready(
+    function ()
+{
+    var a = document.getElementByClassName("check");
+    
+    var ct = document.createElement("span");
+    ct.setAttribute("class","glyphicon");
```

Comment:
looks like you are just setting style to an element, why do you need js for that?